.eslintrc

```diff
@@ -6,6 +6,7 @@
     "plugin:import/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
```

Comment:
Do you need to include it as a plugin to work?

Reply:
It should be enough to extend the configuration unless we need to customize the rules further - https://www.npmjs.com/package/eslint-plugin-react-hooks#installation

Reply:
Gotcha, perfect! I'll approve then.